### PR TITLE
OUT-546 In list view, improve colors, borders, hover states

### PR DESCRIPTION
**Pull request by arpandhakal**
### Tasks

- [In list view, improve colors, borders, hover states](https://linear.app/copilotplatforms/issue/OUT-546/in-list-view-improve-colors-borders-hover-states)

### Changes

- [x] changed padding between title and label in list view TaskRow.tsx.
- [x]  removed pointer cursor in list view rows
- [x]  removed onHover color change for list view rows
- [x]  adjusted height of rows according to figma design

### Testing Criteria

- [LOOM](https://www.loom.com/share/a6f2caf317924fdd8c1a75ce2c47e253)

### CODE REVIEW

File: src/components/cards/ListViewTaskCard.tsx [hunk 1]
```diff
@@ -43,15 +43,14 @@ export const ListViewTaskCard = ({
         className="task-list-card"
         sx={{
           userSelect: 'none',
-          ':hover': {
-            bgcolor: (theme) => theme.color.gray[100],
-          },
+          cursor: 'default',
+          borderBottom: (theme) => `1px solid ${theme.color.borders.borderDisabled}`,
```
Comment (aatbip) [inline src/components/cards/ListViewTaskCard.tsx:47]: hover bgColor is missing

**Review by aatbip — CHANGES_REQUESTED** [review]

File: src/components/cards/ListViewTaskCard.tsx (hunk 1 above)
Comment (arpandhakal) [reply src/components/cards/ListViewTaskCard.tsx:47]: image

hover state is not shown in figma design.

Comment (aatbip) [reply src/components/cards/ListViewTaskCard.tsx:47]: Hover color has been changed I guess but it is there. Please have a look at the image below - 

Screenshot from 2024-07-10 18-46-54

**Comment by aatbip** [comment]
Design of the assignee doesn't match the Figma design - 

Screenshot from 2024-07-11 09-25-10

In our case, we have given bgColor to the assignee selector. 
@arpandhakal

File: src/components/inputs/Selector.tsx [hunk 2]
```diff
@@ -125,9 +125,6 @@ export default function Selector({
               cursor: disabled ? 'auto' : 'pointer',
               borderRadius: '4px',
               padding: '4px 8px',
-              ':hover': {
-                backgroundColor: (theme) => theme.color.gray[100],
-              },
```
Comment (aatbip) [inline src/components/inputs/Selector.tsx:130]: Removing this has removed the background on hover from the assignee dropdown in the sidebar. Can we fix this to what it is like in the production branch now? 
See the SS below - 
Screenshot from 2024-07-11 12-13-10

Comment (aatbip) [reply src/components/inputs/Selector.tsx:130]: While fixing this, can you also add a patch for below issue? 
The assignee avatar icon's letter "S" is not properly placed. May be it needs some padding changes. If it is avatar image then it looks fine though.

File: src/components/cards/ListViewTaskCard.tsx [hunk 3]
```diff
@@ -45,6 +45,9 @@ export const ListViewTaskCard = ({
           userSelect: 'none',
           cursor: 'default',
           borderBottom: (theme) => `1px solid ${theme.color.borders.borderDisabled}`,
+          ':hover': {
+            bgcolor: (theme) => theme.color.gray[150],
```
Comment (aatbip) [inline src/components/cards/ListViewTaskCard.tsx:49]: The bgColor for hover state is not `gray[150]`. Can you please check in Figma once again? 

The actual color is `gray[100]` i.e. `#F8F9FB` with 50% opacity. The equivalent hex color is shown in the video below using Figma color picker. Please follow the same. 
Screenshot from 2024-07-11 12-23-30

Actual hex value for bgColor using Figma picker - 
[color-picker.webm](https://github.com/copilot-platforms/tasks-app/assets/38468429/bfa7b23b-47fb-4312-8c16-9d6d80095cd1)

**Review by aatbip — CHANGES_REQUESTED** [review]
Please fix the bgColor on hover issues below.

File: src/components/cards/ListViewTaskCard.tsx (hunk 3 above)
Comment (aatbip) [reply src/components/cards/ListViewTaskCard.tsx:49]: `#FBFCFD` is the actual hex value.

File: src/components/cards/ListViewTaskCard.tsx [hunk 4]
```diff
@@ -43,15 +43,17 @@ export const ListViewTaskCard = ({
         className="task-list-card"
         sx={{
           userSelect: 'none',
+          cursor: 'default',
+          borderBottom: (theme) => `1px solid ${theme.color.borders.borderDisabled}`,
           ':hover': {
-            bgcolor: (theme) => theme.color.gray[100],
+            bgcolor: (theme) => alpha(theme.color.gray[100], 0.5),
```
Comment (aatbip) [inline src/components/cards/ListViewTaskCard.tsx:49]: this is still being rendered as `#F8F9FB` where as the color in Figma is `#FBFCFD`.

**Review by aatbip — CHANGES_REQUESTED** [review]

**Review by aatbip — APPROVED** [review]
work of an art!

**Pull request merged**